README.md

```diff
@@ -77,6 +77,12 @@ To update the design system templates run:
 make load-design-system-templates
 ```
 
+To download the latest schemas from the questionnaire registry(https://github.com/ONSdigital/eq-questionnaire-schemas):
```

Comment:
I was kind of thinking the **questionnaire registry** text would be the link, sorry